docs/how-to-add-cypress-tests.md

```diff
@@ -0,0 +1,64 @@
+# How to add Cypress tests
+
+When making changes to JavaScript, CSS, or HTML which could change the functionality or layout of a page, it's important to add corresponding [Cypress](https://docs.cypress.io) integration tests.
+
+To learn how to write Cypress tests, or 'specs', please see Cypress' official [documentation](https://docs.cypress.io/guides/getting-started/writing-your-first-test.html).
+
+> Note: When writing tests for freeCodeCamp, remember to add `/* global cy */` to the top of the file to avoid ESLint issues.
+
+### Where to add a test
+
+- Cypress tests are in the `./cypress` directory.
+
+- Cypress tests for a curriculum module are in the corresponding curriculum directory, i.e. `cypress/integration/learn/responsive-web-design/basic-css/index.js`.
+
+### How to run tests
+
+**1. Change directory to the freeCodeCamp directory:**
+
+    ```console
+    cd yourProjectDirectory
+    ```
+
+**2. Start the MongoDB server in a separate terminal**
+
+  - On macOS & Ubuntu:
+
+    ```console
+    mongod
+    ```
+
+  - On Windows, you must specify the full path to the `mongod` binary
+
+    ```console
+    "C:\Program Files\MongoDB\Server\3.6\bin\mongod"
+    ```
```

Comment:
Typically, on Linux at least, `mongod` is going to be running as service and not need manually starting each time.

The main thing is we can reuse the [existing docs](https://contribute.freecodecamp.org/#/how-to-setup-freecodecamp-locally?id=step-3-start-mongodb-and-seed-the-database) here and just remind people that it needs to be running.

Reply:
Thanks for the feedback! Address both concerns in latest commits.